_episodes/03-intro-git-github.md

```diff
@@ -1,15 +1,248 @@
+{: .callout}
+
+
+> ## Line Endings
+>
+> As with other keys, when you hit <kbd>Return</kbd> on your keyboard,
+> your computer encodes this input as a character.
+> Different operating systems use different character(s) to represent the end of a line.
+> (You may also hear these referred to as newlines or line breaks.)
+> Because Git uses these characters to compare files,
+> it may cause unexpected issues when editing a file on different machines. 
+> Though it is beyond the scope of this lesson, you can read more about this issue 
+> [in the Pro Git book](https://www.git-scm.com/book/en/v2/Customizing-Git-Git-Configuration#_core_autocrlf).
+{: .callout}
+>
+> You can change the way Git recognizes and encodes line endings
+> using the `core.autocrlf` command to `git config`.
+> The following settings are recommended:
+>
+> On macOS and Linux:
+>
+> ~~~
+> $ git config --global core.autocrlf input
+> ~~~
+> {: .language-bash}
+>
+> And on Windows:
+>
+> ~~~
+> $ git config --global core.autocrlf true
+> ~~~
+> {: .language-bash}
+> 
+
+
+Riley also has to set his favorite text editor, following this table:
+
+| Editor             | Configuration command                            |
+|:-------------------|:-------------------------------------------------|
+| Atom | `$ git config --global core.editor "atom --wait"`|
+| nano               | `$ git config --global core.editor "nano -w"`    |
+| BBEdit (Mac, with command line tools) | `$ git config --global core.editor "bbedit -w"`    |
+| Sublime Text (Mac) | `$ git config --global core.editor "/Applications/Sublime\ Text.app/Contents/SharedSupport/bin/subl -n -w"` |
+| Sublime Text (Win, 32-bit install) | `$ git config --global core.editor "'c:/program files (x86)/sublime text 3/sublime_text.exe' -w"` |
+| Sublime Text (Win, 64-bit install) | `$ git config --global core.editor "'c:/program files/sublime text 3/sublime_text.exe' -w"` |
+| Notepad++ (Win, 32-bit install)    | `$ git config --global core.editor "'c:/program files (x86)/Notepad++/notepad++.exe' -multiInst -notabbar -nosession -noPlugin"`|
+| Notepad++ (Win, 64-bit install)    | `$ git config --global core.editor "'c:/program files/Notepad++/notepad++.exe' -multiInst -notabbar -nosession -noPlugin"`|
+| Kate (Linux)       | `$ git config --global core.editor "kate"`       |
+| Gedit (Linux)      | `$ git config --global core.editor "gedit --wait --new-window"`   |
+| Scratch (Linux)       | `$ git config --global core.editor "scratch-text-editor"`  |
+| Emacs              | `$ git config --global core.editor "emacs"`   |
+| Vim                | `$ git config --global core.editor "vim"`   |
+| VS Code                | `$ git config --global core.editor "code --wait"`   |
+
+It is possible to reconfigure the text editor for Git whenever you want to change it.
```

Comment:
Should we add a statement recommending `nano` if they haven't installed or used the other options before?